### PR TITLE
Document Directory endpoint

### LINKED ISSUE #3895: GraphQL: Add currency/countries/country queries
# Community engineering issue

Type of Doc:
- [x] New topic
- [ ] Topic Update, Devdoc link:
- [ ] Other

Magento 2 GitHub Issue: https://github.com/magento/graphql-ce/issues/353
Magento 2 GitHub PR: https://github.com/magento/graphql-ce/pull/313

Magento 2 version:
- [ ] 2.1.X
- [ ] 2.2.X
- [x] 2.3.X
- [ ] back port
- [ ] forward port

Notifications of Changes:
- [ ] Changes to Admin interface?
- [ ] New software requirements?
- [x] New REST endpoints or updated GraphQL query syntax?
- [ ] New command line option?

## Content

Queries that mirror the following REST calls have been added for 2.3.1

GET /V1/directory/currency
GET /V1/directory/countries
GET /V1/directory/countries/:countryId

### PR DESCRIPTION
**Pull request by erikmarr**
## This PR is a:

- [x] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will close #3895.

List all affected URLs 

- New graphql topic under the reference section

### CODE REVIEW

File: guides/v2.3/graphql/reference/directory.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
+---
+group: graphql
+title: Directory endpoint
+---
+
+To retrieve country and regional directory information, use the `country`, `countries`, and `currency` endpoints.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/directory.md]:
```suggestion
To retrieve country and regional directory information, use the `country`, `countries`, and `currency` queries.
```

File: guides/v2.3/graphql/reference/directory.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+          "name": "Western Australia"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Countries
+Use the `countries` endpoint to return all countries in which the entity can do business.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/directory.md]:
```suggestion
Use the `countries` query to return all countries in which the entity can do business.
```

File: guides/v2.3/graphql/reference/directory.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+            "id": 95,
+            "code": "WI",
+            "name": "Wien"
+          }
+        ]
+      },
+```
+
+## Currency
+Use the `currency` endpoint to return information about the store's currency configuration.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/directory.md]:
```suggestion
Use the `currency` query to return information about the store's currency configuration.
```

File: guides/v2.3/graphql/reference/directory.md (hunk 1 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/directory.md]: fixed

File: guides/v2.3/graphql/reference/directory.md (hunk 2 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/directory.md]: fixed

File: guides/v2.3/graphql/reference/directory.md (hunk 3 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/directory.md]: fixed

**Comment by dshevtsov** [comment]
@erikmarr
There is missing whatsnew for the Major Update PR.

**Pull request merged**